woocommerce/woocommerce-ios
# Order Creation: Remove feature flag for shipping, fees and remote sync

Closes: https://github.com/woocommerce/woocommerce-ios/issues/6277.

## Description

This PR prepares payment section milestone for release, since functionality for shipping, fees and remote sync is now stable.

## Changes

- Enables remote sync by default
- Removes feature flag check for shipping and fees
- Removes `.orderCreation` feature flag completely
- Updates `PaymentDataViewModel` init to filter out deleted items from `shippingLines` and `fees` (because tests failed with remote synchronizer implementation)

## Testing

1. Build and run the app in debug mode (to ensure the feature flag is enabled).
2. Make sure Order Creation is enabled under Settings > Experimental Features.
3. Go to the Orders tab and tap the + button. Tap "Create order" in the bottom sheet.
4. Check functionality of "Shipping" and "Fees" rows in the "Payment" section.
5. Check that order total is correctly updated after remote sync on every products/shipping/fee update.

---
